.travis.yml

```diff
@@ -13,8 +13,8 @@ matrix:
   include:
     - php: 7.1
       env: dependencies="--prefer-lowest --prefer-stable"
-    - php: 7.2
-      env: dependencies="--prefer-lowest --prefer-stable"
+    - php: 7.3
```

Comment:
why didn't you put it in `php` section as well? Would it be different?

Reply:
I need to add the `--ignore-platform-reqs` for PHP 7.3 (see PR secription) and this is the simplest way imho.